openwisp_network_topology/api/views.py

```diff
@@ -36,7 +77,7 @@ class NetworkGraphView(generics.RetrieveAPIView):
     queryset = Topology.objects.filter(published=True)
 
 
-class ReceiveTopologyView(APIView):
+class ReceiveTopologyView(RequireAuthentication):
```

Comment:
here we don't need it, scripts that send the topology data will use the topology uuid and key as authentication method

Reply:
So Authentication and permission checks should be removed here?

Reply:
> So Authentication and permission checks should be removed here?

yes